### PR TITLE
Onboarding improvements

### PR DESCRIPTION
**Pull request by aantn**

### CODE REVIEW

File: src/robusta/cli/integrations_cmd.py [hunk 1]
```diff
@@ -58,13 +58,56 @@ def get_slack_key() -> (str, str):
...
+        if not try_again:
+            return ""
+
+
+@app.command()
+def ui():
+    ui_key = get_ui_key()
+    if ui_key:
+        log_title(
+            f"Your UI key is:\n{ui_key}\nAdd it to the Robusta sink configuration"
```
Comment (arikalon1) [inline src/robusta/cli/integrations_cmd.py:108]: I would say 'add it as `token` to the Robusta sink configuration'

File: src/robusta/cli/main.py [hunk 2]
```diff
@@ -126,11 +127,7 @@ def write_values_file(output_path: str, values: HelmValues):
     with open(output_path, "w") as output_file:
         yaml.safe_dump(values.dict(exclude_defaults=True), output_file, sort_keys=False)
         typer.secho(
-            f"Saved configuration to {output_path}",
-            fg="green",
-        )
-        typer.secho(
-            f"Save this file for future use. It contains your account credentials",
+            f"Saved configuration to {output_path} - save this file for future use!",
             fg="red",
```
Comment (arikalon1) [inline src/robusta/cli/main.py:131]: Why red?
I think it hints there's an error

In all other places we print red only on error

File: src/robusta/cli/main.py [hunk 3]
```diff
@@ -380,14 +319,17 @@ def gen_config(
...
         ]
-    
-
 
     write_values_file(output_path, values)
 
     if robusta_api_key:
         typer.secho(
-            f"Finish the Helm install and then login to Robusta UI at {backend_profile.robusta_ui_domain}\n",
+            f"Finish installing with Helm (see the Robusta docs). Then login to Robusta UI at https://platform.robusta.dev\n",
```
Comment (arikalon1) [inline src/robusta/cli/main.py:327]: We need to take the url from the backend profile

**Review by arikalon1 — APPROVED** [review]
Looks good
Left few minor comments

File: src/robusta/cli/main.py (hunk 2 above)
Comment (aantn) [reply src/robusta/cli/main.py:131]: I tried doing them both in green but this and the next message just merged together visually. I think it's good this way and it's red as it requires action on your part. It's warning you to save the file.

File: src/robusta/cli/integrations_cmd.py (hunk 1 above)
Comment (aantn) [reply src/robusta/cli/integrations_cmd.py]: I updated it so that it just prints the actual YAML you need. Is that OK?

**Comment by aantn** [comment]
@arikalon1 replied to your comments and pushed an improvement. If it's OK, I'd like to merge.

**Review by arikalon1 — APPROVED** [review]

**Pull request merged**